snowflake/plugin.go

```diff
@@ -41,6 +41,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"snowflake_view":              tableSnowflakeView(ctx),
 			"snowflake_view_grant":        tableSnowflakeViewGrant(ctx),
 			"snowflake_warehouse":         tableSnowflakeWarehouse(ctx),
+			"snowflake_resource_monitor":  tableSnowflakeWarehouse(ctx),
```

Comment:
Can you please put this in alphabetical order?

Reply:
Done